application/src/test/resources/speedtest/travelSearch-expected-results-bdr.csv

```diff
@@ -1,3 +1,3 @@
 tcId,nTransfers,duration,cost,walkDistance,startTime,endTime,agencies,modes,routes,stops,details
-1,0,42m50s,0,0,13:17:10,14:00:00,,,,,Unknown transit 0tx 42m50s
-2,0,42m50s,0,0,13:17:10,14:00:00,,,,,Unknown transit 0tx 42m50s
+1,0,42m47s,0,0,13:17:13,14:00:00,,,,,Unknown transit 0tx 42m47s
+2,0,42m47s,0,0,13:17:13,14:00:00,,,,,Unknown transit 0tx 42m47s
```

Comment:
Do you know why all of these speed tests are 3 seconds quicker? Ideally changing the internal angle representation should not change travel time.

Reply:
It is because of the inAngle / outAngle fix for creating the temporary edge has changed the turn time near the origin / destination.